### ISSUE TITLE
Fix a little bug

### ISSUE THREAD

**Issue by Scuottolinx** [fatturaelettronicaphp/FatturaElettronica#30]
in \FatturaElettronicaPhp\FatturaElettronica\Parser\Body\ShipmentInformationsParser

in row 66 add this
```
if ($value !== null && count($value) > 0) {
          $value = array_shift($value);
```
because the extract return an array

**Comment by Skullbock** [comment]
Hi @Scuottolinx thanks for reporting this.
Can you share a redacted XML file where the current parser fails in that regard?
That way i can add a test for this to avoid regression

Thanks!
 P.s. Feel free to send a PR for this if you can :)

**Comment by Skullbock** [comment]
@Skullbock Found the issue and fixed it in 2.2.3, with dedicated tests
